metagov/metagov/core/views.py

```diff
@@ -274,13 +274,15 @@ def plugin_auth_callback(request, plugin_name):
     plugin_views = importlib.import_module(f"metagov.plugins.{plugin_name}.views")
 
     try:
-        return plugin_views.auth_callback(
+        response = plugin_views.auth_callback(
             type=type,
             code=code,
             redirect_uri=redirect_uri,
             community=community,
             state=state_to_pass,
+            request=request
         )
+        return response if response else HttpResponseRedirect(redirect_uri, {"state": state_to_pass})
```

Comment:
This should use the `redirect_with_params` helper to add the query params. I don't think Django `HttpResponseRedirect` supports adding params like that.

Reply:
Fixed